zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -41,6 +42,21 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
       });
     };
 
+    var checkDownloadingDependencies = function() {
+      var isDownloading = false;
+      for (var setting = 0; setting < $scope.interpreterSettings.length; setting++) {
+        if ($scope.interpreterSettings[setting].status === 'DOWNLOADING_DEPENDENCIES') {
+          isDownloading = true;
+          break;
+        }
+      }
+      if (isDownloading) {
+        $timeout(function() {
+          getInterpreterSettings();
+        }, 2000);
```

Comment:
This timeout period will vary based on the number of dependencies.
Should we have Zeppelin broadcast a message when it is done with the downloading?